conf/hiera.yaml

```diff
@@ -0,0 +1,2 @@
+---
+# Hiera5 Global configuration file.
```

Comment:
Think this should contain:
```
# Hiera 5 Global configuration file

version: 5

# defaults:
#   data_hash: yaml_data
# hierarchy:
#   - name: "Global Overrides"
#     path: "overrides.yaml"
#
```

The config is empty in the sense that it does not define any search. At the same time it communicates the intention that global overrides could be added in a file - suggested "overrides.yaml".

For second opinion: ping @DavidS , what would you like the default to be?